### PR TITLE
Add test case to check memory status

### PR DESCRIPTION
**Pull request by v-fional**

### CODE REVIEW

File: AZ3166/tests/UnitTest/MemoryStatusTest.ino [hunk 1]
```diff
@@ -0,0 +1,49 @@
+test(memory_heap)
```
Comment (ArthurMa1978) [inline AZ3166/tests/UnitTest/MemoryStatusTest.ino]: consider to merge into one test call

Comment (v-fional) [reply AZ3166/tests/UnitTest/MemoryStatusTest.ino]: one test() method means one test scenario, and all test() methods belong to one sketch file that seems there is no necessary to merge into one test()

**Comment by ihere1** [comment]
No need to add a new test file now ?

**Comment by v-fional** [comment]
"MemoryStatusTest.ino" file is a newly  added. As of now, it just contains the heap test.

**Pull request merged**